### PR TITLE
Updating Acculturation Guide

### PR DESCRIPTION
**Pull request by ranjani789**

### CODE REVIEW

File: AcculturationGuide/acculturation.rst [hunk 1]
```diff
@@ -220,13 +228,13 @@ The operation of YottaDB is controlled by a number of environment variables. In
...
    yottadbuser@yottadbworkshop:~$ 
 
-YottaDB databases can also be configured so that they can be recovered after a system crash. Simulate a crash right now by either clicking on the “X” in the top right corner of your virtual machine console window to instantly “power down” your virtual machine, or, if you started it headless, perform a hard power-down using a command on the host (in the case of virtualization using qemu/kvm on Linux, a kill -9 of the virtual machine process). Then reboot the virtual machine, run ydb and use a zwrite ^Disney command to confirm that the data in the database is still intact.
+YottaDB databases can also be configured so that they can be recovered after a system crash. Simulate a crash by either clicking on the “X” in the top right corner of your virtual machine console window to instantly “power down” your virtual machine, or, if you started it headless, perform a hard power-down using a command on the host (in the case of virtualization using qemu/kvm on Linux, a kill -9 of the virtual machine process). Then reboot the virtual machine, run ydb and use a zwrite ^Disney command to confirm that the data in the database is still intact.
```
Comment (ksbhaskar) [inline AcculturationGuide/acculturation.rst]: Don't use ^Disney. That's someone else's brand name and they're fiercely protective of it.

File: AcculturationGuide/acculturation.rst [hunk 2]
```diff
@@ -256,8 +264,8 @@ Note that you may have to install the program 'tree' before running the above co
 With YottaDB, you can write applications that implement international character sets using Unicode or ISO/IEC-10646 (the two standards track each other). Connect to the virtual machine with your terminal emulator configured to support the UTF-8 character set. In a fresh terminal session execute the following (the non-printable characters may look different on your session from the screen here, depending on how your terminal emulator renders them):
 
 .. parsed-literal::
-   yottadbuser@yottadbworkshop:~$ export gtm_chset=UTF-8 LC_CTYPE=en_US.utf8
-   yottadbuser@yottadbworkshop:~$ source /usr/local/lib/yottadb/r120/ydb_env_set
+   yottadbuser@yottadbworkshop:~$ export ydb_chset=UTF-8 LC_CTYPE=en_US.utf8
```
Comment (ksbhaskar) [inline AcculturationGuide/acculturation.rst]: Does YottaDB accept ydb_chset yet?

File: AcculturationGuide/acculturation.rst (hunk 1 above)
Comment (ranjani789) [reply AcculturationGuide/acculturation.rst]: Okay, I will update this example.

File: AcculturationGuide/acculturation.rst (hunk 2 above)
Comment (nars1) [reply AcculturationGuide/acculturation.rst]: Not yet.

Comment (ranjani789) [reply AcculturationGuide/acculturation.rst]: I thought ydb_env_set set it. I can revert it back to gtm_chset.

Comment (nars1) [reply AcculturationGuide/acculturation.rst]: To clarify my response, libyottadb.so/mumps/mupip/dse etc. have no idea of "ydb_chset" as of now (even though ydb_env_set might set this env var). They still look at "gtm_chset".

**Pull request merged**